.github/workflows/build.yml

```diff
@@ -221,6 +221,7 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
       - name: Check & build polkadot docker image
+        if: ${{ needs.set-tags.outputs.git_branch != master }}
```

Comment:
I think this should not be skipped even in master. It ensures the polkadot binary is there so the paratests can run correctly.
@timbrinded do we still need the polkadot binary in para tests or did we move to Moonwall already ?

Reply:
We've moved to moonwall, but that still requires a polkadot binary to run the relay part of the network with. If we want to remove this step then lets refactor the `zombie_upgrade_test` job to get that pull that polkadot binary.

Reply:
> to get that pull that polkadot binary

I think that we still need ti build our own relay node, to test our eventual forks/cherry-picks that are not merged upstream yet